template/src/api/index.js

```diff
@@ -1,7 +1,7 @@
 const Hermes = require('hermesjs');
 const app = new Hermes();
 const path = require('path');
-const { cyan, gray, yellow } = require('colors/safe');
+const chalk = require('chalk');
```

Comment:
```suggestion
const { cyan, gray, yellow } = require('chalk');
```

If you require it like this, you will not have to add `chalk` everywhere :)

Reply:
Done